domain/src/main/java/com/teamagam/gimelgimel/domain/utils/PreferencesUtils.java

```diff
@@ -32,6 +32,10 @@ public void toggleCoordinateSystemPrefs() {
     mUserPreferencesRepository.setPreference(USE_UTM_PREF_KEY, !shouldUseUtm());
   }
 
+  public void setUsername(String username) {
```

Comment:
I actually prefer to delete it and keep the direct implementation.
It's in the domain and used only by the app.